osquery/tables/system/windows/nt_info.cpp

```diff
@@ -0,0 +1,51 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+#include <osquery/status.h>
```

Comment:
Why do you need to explicitly include `status.h` here? I'm asking because I can not see any mention of `class Status` in the code.